README.md

```diff
@@ -75,6 +75,9 @@ operators][operators].
 - https://github.com/giantswarm/release-operator
 - more to come
 
+## Example
+An example `memcached-operator` is provided in the repository [giantswarm/memchached-operator](https://github.com/giantswarm/memcached-operator). We try to keep this operator up-to-date and for it to be used as the 'state of the art' example.
```

Comment:
We agreed that `aws-operator` is our SotA Operator. The `memcached-operator` is rather a simplified example to illustrate certain primitives.